### ISSUE TITLE
incorrect events matched

### ISSUE THREAD

**Issue by AndrewLangvt** [timbitz/Whippet.jl#65]
Hi Tim, 

Finally cranking through this analysis with my entire dataset now.  Running into an issue with "matching events" in the whippet_delta function. 

```
WARNING: Method definition count() in module IterTools at deprecated.jl:49 overwritten in module Iterators at deprecated.jl:49.
WARNING: Method definition count(Number) in module IterTools at deprecated.jl:49 overwritten in module Iterators at deprecated.jl:49.
WARNING: Method definition count(Number, Number) in module IterTools at deprecated.jl:49 overwritten in module Iterators at deprecated.jl:49.
Whippet v0.8 loading and compiling... 
 6.326671 seconds
Sample A: L-Blu13_male_gonad_control.NYNO.psi.gz,L-G107_male_gonad_control.psi.gz,L-R3_male_gonad_control.NYNO.psi.gz,L-R8_male_gonad_control.psi.gz,L-W33_male_gonad_control.psi.gz,L-W3_male_gonad_control.NYNO.psi.gz,L-W4_male_gonad_control.NYNO.psi.gz,R-Y108-W29_male_gonad_control.psi.gz,x-blk-blk-ATLAS_male_gonad_control.psi.gz,x-g-ATLAS_male_gonad_control.psi.gz,x-g-g-ATLAS_male_gonad_control.psi.gz,x-g-g-g-ATLAS_male_gonad_control.psi.gz,x-y-s-ATLAS_male_gonad_control.psi.gz
Sample B: g105-x_male_gonad_stress.psi.gz,L-Blu123-W38_male_gonad_stress.psi.gz,L-O116_male_gonad_stress.psi.gz,L-O123-W28_male_gonad_stress.psi.gz,L-R2_male_gonad_stress.psi.gz,L-Y101_male_gonad_stress.psi.gz,L-Y111_male_gonad_stress.psi.gz,o122-g2-x_male_gonad_stress.psi.gz,o124-y124-x_male_gonad_stress.psi.gz,R-Blu8-W30_male_gonad_stress.psi.gz,R-G124_male_gonad_stress.psi.gz,R-O106-W27_male_gonad_stress.psi.gz,y110-x_male_gonad_stress.psi.gz
Now processing files and calculating posterior distributions...
ERROR: LoadError: AssertionError: Incorrect events matched!!
 in #process_psi_files#90(::Int64, ::Int64, ::Float64, ::Int64, ::Function, ::String, ::Array{BufferedStreams.BufferedInputStream,1}, ::Array{BufferedStreams.BufferedInputStream,1}) at /opt/packages/whippet/Whippet.jl-0.8/src/diff.jl:125
 in (::Whippet.#kw##process_psi_files)(::Array{Any,1}, ::Whippet.#process_psi_files, ::String, ::Array{BufferedStreams.BufferedInputStream,1}, ::Array{BufferedStreams.BufferedInputStream,1}) at ./<missing>:0
 in macro expansion at /opt/packages/whippet/Whippet.jl-0.8/src/timer.jl:5 [inlined]
 in main() at /opt/packages/whippet/Whippet.jl-0.8/bin/whippet-delta.jl:91
 in include_from_node1(::String) at ./loading.jl:488
 in process_options(::Base.JLOptions) at ./client.jl:265
 in _start() at ./client.jl:321
while loading /opt/packages/whippet/Whippet.jl-0.8/bin/whippet-delta.jl, in expression starting on line 99
```

I have tried running this command exactly the same, except with removing each psi.gz file, one at a time, thinking perhaps one of my samples didn't map properly.  Still get the same error though... any thoughts as to what might be the underlying issue?

**Comment by timbitz** [comment]
Hi @AndrewLangvt, It looks like you're still using Whippet v0.8?.. Can you update to v0.10.2? **a lot** has changed since then, and it is not recommended to use such an old version.

That being said, I did just find a bug that was outputting an extra dummy PSI line for tandem 3'UTRs (which sounds like it could throw such an error in theory) that I have fixed in #64, but I haven't yet merged these changes, though I will in the next couple days. 

Either way, all Whippet versions from v0.9+ are now installed as a standard package from the julia REPL making it trivial to update to the newest releases in the future by using 'Pkg.update()'.

**Comment by AndrewLangvt** [comment]
I figured it out, not sure what the bug is, though likely fixed by a version update.  I was able to still run my wippet_delta comparisons so long as I ran it from the directory where the quant files are located. Seems like running from another directory, even with the `-d` flag specifying where to look for the `psi.gz` files was throwing some issue.  Thanks for the quick response!
